src/pages/css/properties/color-property/index.md

```diff
@@ -1,15 +1,47 @@
----
```

Comment:
Please restore this frontmatter, we use this title for the UI 👍

Reply:
Okay. Will put it back in.

Reply:
Did that fix it?

Reply:
@arhillis I think you have just edited you first post on this PR.

Try clicking the `files changed` tab and then editing the file there by clicking on the pencil